src/utils.js

```diff
@@ -0,0 +1,38 @@
+export const cleanMovies = (movies) => {
+    return movies.map(movie => {
+        return {
+            id: movie.id,
+            title: movie.title,
+            poster_path: movie.poster_path
+        }
+    })
+}
+
+export const cleanMovie = (selectedMovie) => {
+    let {backdrop_path, title, average_rating,
+        release_date, overview, genres, budget,
+         revenue, runtime, tagline} = selectedMovie;
+    average_rating = average_rating.toFixed(2);
```

Comment:
I also wonder if some of this should be in a separate function?

Reply:
I believe this is fine